content/blog/intro-to-web-accessability/index.md

```diff
@@ -82,7 +81,7 @@ Meeting AA requirements is typically seen as a good commitment to accessibility,
 Far from a comprehensive list, A requires:
 
 - [Non-text content to have alternative text](https://www.w3.org/TR/WCAG21/#non-text-content)
-- [All content accessible with keyboard](https://www.w3.org/TR/WCAG21/#keyboard)
```

Comment:
Removing keyboard from level A as well?

Reply:
Yeah, this was intentional to avoid confusion with the mention of keyboard access for AAA